### ISSUE TITLE
Bug - experiment naming & export

### ISSUE THREAD

**Issue by branscombj** [lzim/teampsd#1350]
First issue is just for facilitators to note. I do not think it requires a change. 2nd issue may be a bug. @lijenn I am not sure how to complete step 2 below.

1. In the Save Experiment section, the value range is independent of the variable selected - it does not reflect the actual range available for the chosen variable. Found this when I ran an experiment in PSY with all variables set to their max, and entered the largest value (200) for each variable in the name - where some have a max possible of "full", 30, 50, or 100.

2. No matter which way I export (Copy/Paste as in 1st screenshot, or Export as in 2nd) the run data to Excel, the Current Run Decision Variables title shows up without any values for the run showing. (This was the run with all exp variables at their max.

![image](https://user-images.githubusercontent.com/29778290/79906608-1d914d80-83e6-11ea-907d-730645e3f93b.png)
![image](https://user-images.githubusercontent.com/29778290/79906640-2c780000-83e6-11ea-9425-7433fd3398ef.png)
 
**2. Click on the right:**  
i. **Pipelines** - prioritize in a TeamPSD (& Workgroup) pipeline  
ii. **Assignee** - assign to an individual team member producing work for the issue  
iii. **Label**- assign workgroup (lead) responsible for tracking the task  
iv. **Projects** - assign to domain-specific TeamPSD tracker (issues, features, document, manuscript, team)  
v. **Milestones** - select monthly sprint milestone within the workgroup   
vi. **Estimate** - Estimate task in complexity points   
vii. **Epics** - assign to team-wide epic and TeamPSD master plan project  
viii. **Releases** - assign to key team release grouped with other relevant issues  

**3. Once you hit "Submit new issue**, underneath the first comment:   
viiii. **Add dependency** - assess for and add dependencies *blocking* & *blocked by* this issue

**Comment by lijenn** [comment]
@branscombj Hm, I've never actually exported the data out before so I'm not too versed in how it should it show up outside of the SIM. 

I tested it myself and I believe I am seeing all the run data?

The run data just doesn't show up under the "Current Run Decision Variables" title. If anything, that just seems like a general header for all the variables shown below.

I am seeing the run data being shown under the "Model Variables" header after I copied Copy/Pasted it:

<img width="964" alt="Screen Shot 2020-04-22 at 10 19 15 AM" src="https://user-images.githubusercontent.com/59668647/80012856-d311de80-8482-11ea-80c8-ab60007281f4.png">

Did I address the problem you were having in Step 2?

**Comment by branscombj** [comment]
Thanks, @lijenn ! I think I did get proper run data; it's the Current Run Decision Variables should show what variable sliders were adjusted for the experiment. They would be inputs, and not show up in a row with different values for each week.
@jamesmrollins Do you know if I'm doing something wrong or looking at the output wrong?
@dlkibbe fyi.

**Comment by jamesmrollins** [comment]
Hi @branscombj , I'm not sure I understand the problem.  Are you saying the experiment variables are showing as different for each week?  If so, that would be a problem.  Could you please upload the .xlsx file in question to this thread so I can take a look?

**Comment by branscombj** [comment]
Hi @jamesmrollins - Can you see in the 3 screenshots above (2 from me and one from @lijenn ) how there is nothing aftr the row heading "Current Run Decision Variables"? I confess that I did not enter any QHFD text, so I'm not surprised those are blank; but I did adjust experiment sliders at least in the one my excel screenshots are showing.

**Comment by jamesmrollins** [comment]
@branscombj @lijenn so you are in PROD, right?

**Comment by jamesmrollins** [comment]
@branscombj and @lijenn I think you need to scroll down further to see your experiment settings.  Perhaps we should just remove the line, "Current Run Decision Variables."

![image](https://user-images.githubusercontent.com/30132017/80015549-c68f8500-8486-11ea-9664-2aba6256de96.png)

**Comment by branscombj** [comment]
> @branscombj and @lijenn I think you need to scroll down further to see your experiment settings. Perhaps we should just remove the line, "Current Run Decision Variables."
> 
Ah, yes, I see. I thought in my dim memory that the settings showed up at the top, sort of the way they do in the Control Panel. 

Yes, I think removing that line would be a good idea.

**Comment by jamesmrollins** [comment]
@branscombj @lijenn we could also move the experiment settings to the header, just say the word.

**Comment by branscombj** [comment]
> @branscombj @lijenn we could also move the experiment settings to the header, just say the word.

Can just the altered variables and their settings be shown, and only once (not across all the weeks of the run)? If so, that would be my vote. But I feel I'm raising something that is outside the current Epic and don't want to derail your priority work!  I don't know if this issue has even been assigned to a kanban column?

**Comment by jamesmrollins** [comment]
> > @branscombj @lijenn we could also move the experiment settings to the header, just say the word.
> 
> Can just the altered variables and their settings be shown, and only once (not across all the weeks of the run)? If so, that would be my vote. But I feel I'm raising something that is outside the current Epic and don't want to derail your priority work! I don't know if this issue has even been assigned to a kanban column?

I don't think removing each week's experiment setting values is a good idea.  We are currently running under the paradigm of a full 2-year run.  Eventually, we may start making 1 year course correction experiments where the value change would be reflected.  

You are also correct, that if we change the export, we should put it into the process the proper way to account for the work.  I'm still shaking off the "Wild West" development environment we have used to date 😁.

**Comment by branscombj** [comment]
GOT IT re: Eventually, we may start making 1 year course correction experiments where the value change would be reflected.
Thanks for that reminder.

@lijenn , would you be able to triage this issue to where it needs to go in our workflow? I think it's now simply the bottom line question,

**Should we remove the header row that says "Current Run Decision Variables" from both the COPY run data output and the EXPORT run data output?**  
Reasoning: 
a) That data is not part of the COPY export, and
b) It is found in the EXPORT file by scrolling further down below the output called "Model Variables" to the section called "Experiment Variables".

p.s. If I had my druthers, I might a) include it in the COPY output; and b) call those two sections, respectively, **Experiment Results** and **Experiment Settings**.

**Comment by dlkibbe** [comment]
Thanks for fyi'ing me on this @branscombj 
I agree the phrase "Current Run Decision Variables" could be removed.
The titles in the p.s. would be logical to a learner in my opinion. I wonder about the **Experiment Settings** appearing at the bottom of the file vs. the top of the file so the learner instantly recognizes what they entered into the model but that's not really important.

**Comment by branscombj** [comment]
> I wonder about the Experiment Settings appearing at the bottom of the file vs. the top of the file so the learner instantly recognizes what they entered into the model but that's not really important.

I had the same thought, @dlkibbe .

**Comment by dlkibbe** [comment]
@jamesmrollins suggested the movement of settings to the header so I don't think my comment was too far off :-)

**Comment by branscombj** [comment]
Hi @jamesmrollins ! What's happening on this now; anything I need to do? I got an "assigned to" ping.

**Comment by jamesmrollins** [comment]
Hi @branscombj - we are reviewing assignees and adding Development/Operations/Research tabs to provide David with necessary data structure for economic analysis of our efforts. So expect a few of these to come.

**Comment by jamesmrollins** [comment]
@lijenn @mnallajerla I double-checked to see if this was ever completed.  Indeed, it was not.  It is a very easy fix and we should prioritize it.

![image](https://user-images.githubusercontent.com/30132017/151416995-ff64c55d-2fca-43b0-9be7-d8bb12b9d451.png)
